text/0308-cli-advisories.md

```diff
@@ -0,0 +1,263 @@
+```
+
+You can suppress all warnings per individual execution:
+
+```
+$ cdk deploy --no-advisories
+```
+
+And you can disable all advisories indefinitely by adding this entry to
+`~/.cdk.json`:
+
+```
+"supressAllAdvisories": true
+```
+
+Regardless of the state of this flag and the advisories you have acknowledged,
+you can always show the currently active advisories:
+
+```
+$ cdk advisories
+```
+
+This command returns zero if there is no advisory and non-zero otherwise. Users
+can then plug this into a pipeline approval workflow and expect manual review if
+there are any advisories.
+
+```
+[ ] Signed-off by API Bar Raiser @xxxxx
+```
+
+## Public FAQ
+
+### What are we launching today?
+
+A new communication channel between AWS and users of the CDK. Starting on
+version x.y.z of the CDK CLI, customers will be notified, on every command,
+about security vulnerabilities, regressions and usage of unsupported versions.
+
+### Why should I use this feature?
+
+These advisories shown by the CLI contain very important and actionable
+information about problems that directly affect you. They give you an
+opportunity to upgrade the CLI or the construct library when necessary or to
+work around high-impacting issues.
+
+## Internal FAQ
+
+### Why are we doing this?
+
+In case of emergency announcements, such as security vulnerabilities or
+regressions, the only mechanisms we have right now are email campaigns and
+pinned GitHub issues. These are not always the best means to reach out to
+customers. Many email addresses are not monitored by anyone and customers don’t
+necessarily check the GitHub repository for updates. The output of the CLI, on
+the other hand, is seen by many customers every day.
+
+### Why should we *not* do this?
+
+This is a powerful feature to convey urgent and important messages to customers.
+However, to keep its relevance, it should be used sparingly and the messages
+must meet a high bar. Otherwise it will just become a tool for spamming
+customers. We will introduce filters to reduce this risk, by only showing
+content that applies to each particular environment. But ultimately, it hinges
+on responsible use. In particular, this feature should not be used for things
+like marketing campaigns, blog post announcements or upcoming events. If we
+don’t have a good mechanism for defining and enforcing this constraint, we
+should not implement this feature.
+
+### What is the technical solution (design) of this feature?
+
+To publish a new advisory, all that the CDK team will have to do is create a
+GitHub issue that fulfils some requirements. The CLI will consume these issues
+using the GitHub API and apply a set of filters to narrow down the list of
+advisories to the context in which it is being run. To improve the publishing
+experience, we will implement a GitHub action that will validate whether the
+issues that are candidates for advisories fulfil the requirements. For a more
+detailed explanation, see the Appendix.
```

Comment:
See above - instead of appendix, describe the publisher experience in an additional "working backwards" section.